webview/src/experiments/components/table/styles.module.scss

```diff
@@ -461,6 +422,11 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
     display: flex;
     flex-flow: row nowrap;
     text-align: left;
+
+    &:hover {
```

Comment:
Do we want our rows to be highlighted on hover similar to the column hover styles? 

<img width="235" alt="image" src="https://user-images.githubusercontent.com/43496356/182492586-bce2a9f0-291c-4d61-bde7-133ab3994699.png">

Reply:
In this case, let's try to stick to the plan as much as possible. Thanks